tests/snmp/test_snmp_phy_entity.py

```diff
@@ -630,7 +630,7 @@ def test_turn_off_psu_and_check_psu_info(duthosts, enum_rand_one_per_hwsku_hostn
     :param pdu_controller: PDU controller
     :return:
     """
-    duthost = duthosts[enum_rand_one_per_hwsku_hostname]
+    duthost = duthosts[enum_supervisor_dut_hostname]
```

Comment:
The change seems ok.
Shouldn't skip handle this scenario, why do we need to run only on supervisor?

Reply:
old behavior should also be fine.
old:
for chassis it choose sup and cards, but cards were skipped
new:
for chassis it choose sup only

the only con is we can avoid some `skipped` tests in pipeline result.